Sync EdgeZero PR #257 updates (#761)
IABTechLab/trusted-server: docs/guide/integration-guide.md

### 7. Provide Static Assets (If Needed)

> Place any integration-specific JavaScript entrypoint under `crates/trusted-server-js/lib/src/integrations/` (for example, `crates/trusted-server-js/lib/src/integrations/testlight.ts`). The shared `npm run build` script automatically discovers every file in that directory and produces a bundle named `tsjs-<entry>.js`, which the Rust crate embeds as `/static/tsjs=tsjs-<entry>.min.js`.

Reviewer: 🔧 **Docs: match the actual JS entrypoint discovery shape**

The build discovers integration directories with an `index.ts` entrypoint (`crates/trusted-server-js/lib/build-all.mjs`), not arbitrary `.ts` files directly under `src/integrations`. Following the current example would create `testlight.ts`, which is not bundled.

```suggestion
Place any integration-specific JavaScript entrypoint under `crates/trusted-server-js/lib/src/integrations/<integration-id>/index.ts` (for example, `crates/trusted-server-js/lib/src/integrations/testlight/index.ts`). The shared `npm run build` script automatically discovers every integration directory with an `index.ts` file and produces a bundle named `tsjs-<integration-id>.js`, which the Rust crate embeds as `/static/tsjs=tsjs-<integration-id>.min.js`.
```

Author: Fixed in 32f45d5b. Updated the section to describe directory-based entrypoint discovery — `crates/trusted-server-js/lib/src/integrations/<integration-id>/index.ts` — matching how `build-all.mjs` actually discovers integrations, and producing `tsjs-<integration-id>.js`.

**Key files**:

- `crates/trusted-server-core/src/integrations/testlight.rs` - Rust implementation

> - `crates/trusted-server-js/lib/src/integrations/testlight.ts` - TypeScript shim

Reviewer: 🔧 **Docs: point at the real Testlight TS entrypoint**

`testlight.ts` does not exist after the directory-entrypoint build layout; the actual bundle entry is `testlight/index.ts`.

```suggestion
- `crates/trusted-server-js/lib/src/integrations/testlight/index.ts` - TypeScript shim
```

Author: Fixed in 32f45d5b. The Testlight key file now points at `crates/trusted-server-js/lib/src/integrations/testlight/index.ts`.
